src/backend/distributed/commands/alter_table.c

```diff
@@ -1967,6 +2040,19 @@ ExecuteQueryViaSPI(char *query, int SPIOK)
 }
 
 
+/*
+ * ExecuteAndLogQueryViaSPI is a wrapper around ExecuteQueryViaSPI, that logs
+ * the query to be executed, as DEBUG1.
+ */
+void
+ExecuteAndLogQueryViaSPI(char *query, int SPIOK)
```

Comment:
Let's accept a log level to function and pass it to the ereport, so that function can be reused for more cases.